src/__tests__/TuxMutableClass-test.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var moduleToTest = '../TuxMutableClass';
+
+jest.mock('react');
+jest.dontMock(moduleToTest);
+
+describe('TuxMutableClass', function () {
+  var React, createMutableClass, tuxMutableClass, mockMutableClassProps, mockMixins, mockConnectMutableToStore, mockStoreMixinGenerator, mockStoreMixin, mockGetMutablePropsMixin, someMutableMockProps;
```

Comment:
Remove unnecessary variable declarations (`mockConnectMutableToStore`, `mockStoreMixinGenerator`, `mockStoreMixin`).  Also, the wrong variables are being declared here with regards to `mockGetOwnerPropsMixin` (being declared as `mockGetMutablePropsMixin`) and `mockMutableClassProps` (being declared as `someMutableMockProps`).